apps/console/src/components/pages/protected/evidence/evidence-create-form.tsx

```diff
@@ -0,0 +1,338 @@
+                                  <InfoIcon size={14} className="mx-1 mt-1" />
+                                </TooltipTrigger>
+                                <TooltipContent side="right">
+                                  <p>Provide a name for the evidence, generally should include the related Control or Task.</p>
+                                </TooltipContent>
+                              </Tooltip>
+                            </TooltipProvider>
+                          </div>
+                          <FormControl>
+                            <Input variant="medium" {...field} className="w-full" />
+                          </FormControl>
+                          {form.formState.errors.name && <p className="text-red-500 text-sm">{form.formState.errors.name.message}</p>}
+                        </FormItem>
+                      )}
+                    />
+                  </InputRow>
+
+                  {/* Description Field */}
+                  <InputRow className="w-full">
+                    <FormField
+                      control={form.control}
+                      name="description"
+                      render={({ field }) => (
+                        <FormItem className="w-full">
+                          <div className="flex items-center">
+                            <FormLabel>Description</FormLabel>
+                            <TooltipProvider disableHoverableContent>
+                              <Tooltip>
+                                <TooltipTrigger type="button">
+                                  <InfoIcon size={14} className="mx-1 mt-1" />
+                                </TooltipTrigger>
+                                <TooltipContent side="right">
+                                  <p>Provide a short description of what is contained in the files or linked URLs.</p>
+                                </TooltipContent>
+                              </Tooltip>
+                            </TooltipProvider>
+                          </div>
+                          <FormControl>
+                            <Textarea id="description" {...field} className="w-full" />
+                          </FormControl>
+                          {form.formState.errors.description && <p className="text-red-500 text-sm">{form.formState.errors.description.message}</p>}
+                        </FormItem>
+                      )}
+                    />
+                  </InputRow>
+
+                  {/* Tags Field */}
+                  <InputRow className="w-full">
+                    <FormField
+                      control={form.control}
+                      name="tags"
+                      render={({ field }) => (
+                        <FormItem className="w-full">
+                          <FormLabel>Tags</FormLabel>
+                          <FormControl>
+                            <MultipleSelector
+                              placeholder="Choose existing or add tag..."
```

Comment:
Lets change the placeholder for now since you can't choose existing ones
```suggestion
                              placeholder="add tags..."
```